optimade/server/entry_collections/mongo.py

```diff
@@ -32,6 +32,11 @@
 
 
 class MongoCollection(EntryCollection):
+    """ Class for querying MongoDB collections (implemented by either pymongo or mongomock)
+    containing serialized [`EntryResource`][optimade.models.EntryResource]s objects.
```

Comment:
Same here as for the comment above. But since writing that comment I am becoming more and more convinced I will fix this in #430 instead of here :)